akka-http-core/src/main/scala/akka/http/javadsl/Http.scala

```diff
@@ -185,29 +209,11 @@ class Http(system: ExtendedActorSystem) extends akka.actor.Extension {
    *
    * The server will be bound using HTTPS if the [[ConnectHttp]] object is configured with an [[HttpsConnectionContext]],
    * or the [[defaultServerHttpContext]] has been configured to be an [[HttpsConnectionContext]].
-   */
-  def bindAndHandle(
```

Comment:
These were introduced for 10.2.0 but are not necessary any more, so I removed them directly.